tool.go

```diff
@@ -0,0 +1,112 @@
+package codacytool
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/josemiguelmelo/gofile"
+	logrus "github.com/sirupsen/logrus"
+	"log"
```

Comment:
`logrus` and `log` one or the other, please be consistent in the usage of the logging library

Reply:
Nice catch! Used `logrus` as logging library